docs/docs/kds-rest-api.md

```diff
@@ -223,25 +212,28 @@ field will contain the API version used for the request (in this case, 1).
 ```
 
-This data, along with the `dataSigningKey` contained in the `kdsAttestationReport` from the response, and the `signature` field are
-used to check the response integrity.
+The request includes the name, the master key type, and the policy constraint. The public key comes from the response 
+after 
+being decoded from Base64. The API version field contains the API version used for the request.
+
+You can use this data, along with the `dataSigningKey` contained in the `kdsAttestationReport` from the response, and 
+the `signature` field to check the response integrity.
+
+The API response holds the public key in 44 bytes. The first 12 bytes represent the algorithm identifier as per 
+[RFC-8410](https://datatracker.ietf.org/doc/html/rfc8410#page-3). The remaining 32 bytes hold the actual public key 
+data. 
 
-On the API response, the public key is represented by 44 bytes: the first 12 of them being the algorithm identifier,
-conforming to [RFC-8410](https://datatracker.ietf.org/doc/html/rfc8410#page-3), and the remaining 32 representing the public key data. 
-On some libraries, when importing the key those bytes representing the algorithm identifier are not taken in consideration,
-resulting in failure on the key import. This can be handled in two ways: firstly, slicing the byte array to ignore the first
-12 bytes and just use the 32 bytes corresponding to the key, or implement a more thorough validation, checking the algorithm 
-identifier to ensure it is supported.
+The key import fails if a library doesn't handle the initial bytes with the algorithm identifier. You can prevent 
+this failure in two ways:
 
-!!! warning
+* Slice the byte array, ignore the first 12 bytes, and use the last 32 bytes to read the key.
+* Check the first 12 bytes to know if the algorithm is supported, and use the last 32 bytes to read the key.
 
-    The integrity check ensures that the public key has not been tampered with or altered inside the response. But it does not guarantee
-    the authenticity of the sender, i.e., the integrity check does not guarantee that the message was sent by the KDS. To ensure the authenticity
-    of the sender, the `kdsAttestationReport` field in the response must be validated.
+The integrity check ensures that no one has tampered with the public key inside the response. But it 
+does not guarantee that the KDS sent the message. To ensure the sender's authenticity, you must validate 
+the `kdsAttestationReport` field in the response.
 
-!!! tip
+!!!Note
 
-    Instead of using the `/public` end point to retrieve KDS public keys to encrypt mails for the enclaves, 
-    it is recommended Java and Kotlin developers use the
-    [`post office`](api/-conclave%20-core/com.r3.conclave.client/-post-office-builder/using-k-d-s.html). This will perform 
-    all the checks for you.
+    Java and Kotlin developers can use [`post office`](api/-conclave%20-core/com.r3.conclave.client/-post-office-builder/using-k-d-s.html)
```

Comment:
Missing "the" here

Reply:
Fixed in [this PR](https://github.com/R3Conclave/conclave-core-sdk/pull/50).